images/scripts/rhel.setup

```diff
@@ -107,6 +107,7 @@ echo foobar | passwd --stdin root
 COCKPIT_DEPS="\
 device-mapper-multipath \
 firewalld \
+gcc \
```

Comment:
FWIW, this still feels wrong/hackish to me. Packages should be built inside mock, not the main VM image. And sub-man's test/vm.install already installs a few extra packages, so it could just as well install gcc and any other stuff that it needs.

Reply:
Ah, sure, PRs are welcome ;) Is gcc the only package missing?